<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AboutEpsonScan2Dialog</class>
 <widget class="QDialog" name="AboutEpsonScan2Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>364</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VERSION_001</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons/icon.qrc">
    <normaloff>:/escan2_app.ico</normaloff>:/escan2_app.ico</iconset>
  </property>
  <widget class="QTableWidget" name="Model_List_tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>341</width>
     <height>121</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectItems</enum>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <property name="gridStyle">
    <enum>Qt::SolidLine</enum>
   </property>
   <property name="sortingEnabled">
    <bool>false</bool>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <property name="columnCount">
    <number>2</number>
   </property>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>16</number>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <column>
    <property name="text">
     <string>LINUX_VERSION_001</string>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>VERSION_002</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="Close_pushButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>260</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>TC_PREVIEW_001</string>
   </property>
  </widget>
  <widget class="QLabel" name="Copyright_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string notr="true">Copyright (C) Seiko Epson Corporation 2018. 
All rights reserved.</string>
   </property>
  </widget>
  <widget class="QLabel" name="Locale_label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>55</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true">EN-US</string>
   </property>
  </widget>
  <widget class="QToolButton" name="Help_toolButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>16</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="toolTip">
    <string notr="true"/>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="icon">
    <iconset resource="icons/icon.qrc">
     <normaloff>:/escan2_app.ico</normaloff>:/escan2_app.ico</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
   <property name="popupMode">
    <enum>QToolButton::DelayedPopup</enum>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
   <property name="arrowType">
    <enum>Qt::NoArrow</enum>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="icons/icon.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>Close_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>AboutEpsonScan2Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>309</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>181</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
